.github/workflows/assign-for-verification.yml

```diff
@@ -1,26 +1,28 @@
-name: Verify Install
-on:
-  issues:
-    types: [labeled]
-jobs:
-  assign:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/github-script@v4
-        with:
-          script: |
-            const label = context.payload.label
-            if (label && label.name === "3 - installed") {
-              github.issues.createComment({
-                issue_number: context.issue.number,
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                body: "This issue is installed, assigning to @benelan for verification",
-              });
-              await github.issues.update({
-                issue_number: context.issue.number,
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                assignees: ["benelan"]
-              });
-            }
+name: Verify Install
+on:
+  issues:
+    types: [labeled]
+jobs:
+  assign:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v4
+        with:
+          script: |
+            const { label, sender } = context.payload;
+            if (sender && label && label.name === "3 - installed") {
+              const assignees =
+                sender.login !== "benelan" ? ["benelan"] : ["jcfranco", "julio8a"];
```

Comment:
Not for this PR and more of an idea, but it'd be useful to have some sort of `VERIFIERS` variable for this and possibly shared between actions. Maybe as GH ssssssssssecrets. 🤫

Reply:
@jcfranco Good idea, if I end up needing to use our usernames in another action I'll make that change.